src/dune

```diff
@@ -20,6 +20,17 @@ let preprocess =
  %s
  (libraries cmdliner ocamlformat_support import ocaml-migrate-parsetree unix))
 
+(rule
+  (targets ocamlformat.1)
+  (deps ocamlformat.exe)
+  (action (with-stdout-to %%{targets} (run ./ocamlformat.exe --help=groff))))
+
+(rule
+  (targets ocamlformat-help.txt)
```

Comment:
Will dune flip its shit if this is `../ocamlformat-help.txt` so that the docs aren't in the source dir?

Reply:
`../ocamlformat-help.txt` will probably not work. but we can move that rule to the parent directory.